README.md

```diff
@@ -133,7 +133,7 @@ We outline our process below to clarify the roles of everyone involved.
 
 All pull requests must be approved by two committers before being merged into
 the repository. If any changes are necessary, the team will leave appropriate
-comments requesting changes to the code. Unfortunately we cannot guarantee a
+comments requesting changes to the code. Unfortunately, we cannot guarantee a
```

Comment:
According to [this discussion](https://english.stackexchange.com/questions/418536/comma-after-unfortunately-in-the-beginning-of-a-sentence), the comma does not seem to be required.

Reply:
@josevalim Ok I will close the pull request.